### PR TITLE
Status

### PR DESCRIPTION
**Pull request by maheshreddy7797**
Enhanced the mayaserver status with hardstuff details
mayaserver_status

### CODE REVIEW

File: cmd/status_test.go [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package cmd
+
+import (
+	"testing"
+
+	"github.com/mitchellh/cli"
+)
+
+func TestStatussCommand_implements(t *testing.T) {
```
Comment (AmitKumarDas) [inline cmd/status_test.go]: Write more test conditions.
Test postive, negative & boundary conditions for below methods:
- Synopsis
- Run
- Help

Comment (maheshreddy7797) [inline cmd/status_test.go]: TeststatusCommand_postive;TeststatusCommand_negative are these the function i need to add??

Comment (AmitKumarDas) [inline cmd/status_test.go]: These are just function names. You can do it.
Please review some bets practices for unit testing golang code & proceed accordingly.

In addition, put appropriate comments in your code.

File: cmd/status_test.go [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+//The code written here will always PASS go test
+
+func TestStatusCommand_Test(t *testing.T) {
+	// Initialize expected arguments as hard stuff
+	expectedport := "5656"
+	expectedip := "172.28.128.4"
+
+	//Initialize actual arguments
+	actualport := "5656"
```
Comment (AmitKumarDas) [inline cmd/status_test.go]: The actual cannot be hardcoded. You need to try getting it from the function that is to be tested.

File: cmd/status.go [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+`
+	return (helpText)
+}
+
+func (c *StatusCommand) Run(_ []string) int {
+	// StatusCommand is a Command implementation prints the Status.
+
+	// Print the text in an allignment
+
+	fmt.Printf("%-18s%-18s%-18s%-18s", "Service", "IP", "Port", "Status")
```
Comment (AmitKumarDas) [inline cmd/status.go]: You need to use c.Ui for printing.

File: cmd/status_test.go (hunk 2 above)
Comment (maheshreddy7797) [inline cmd/status_test.go]: In that case i need create an arguments for port,ip in status file and pass them to test function. But yesterday you said we should not pass any value to arguments in hardcoded stuff

**Comment by maheshreddy7797** [comment]
make_bin

The code has built successfully in my machine

File: cmd/status_test.go (hunk 2 above)
Comment (AmitKumarDas) [inline cmd/status_test.go]: Nothing is passed to test function.

The test function will test the original function.
The original function needs to be passed arguments if original accepts arguments.
The result of original function needs to be checked as actual o/p. In cases where, original function does not return any output, the struct etc needs to be verified. 

In this case, you need to find out the way to extract values from c.Ui as original function has set the output into c.Ui.

You need to refer to various test cases both in maya api server, nomad & other blogs etc to get a convenient solution.

**Comment by maheshreddy7797** [comment]
The errors in 451 and 452 are solved in my case by commiting code two or more times

**Comment by maheshreddy7797** [comment]
sts

**Comment by maheshreddy7797** [comment]
I got the real stuff port and up but...Can't able to slice port number from struct

**Comment by maheshreddy7797** [comment]
The above screen shot is the output with real stuff
